packages/client/src/links/internals/httpUtils.ts

```diff
@@ -133,41 +136,79 @@ export const jsonHttpRequester: Requester = (opts) => {
   });
 };
 
+export const streamingJsonHttpRequester: Requester<
```

Comment:
Could we move this to the `httpBatchStreamLink` rather than being a shared util?